File: general/oop/as03/Ex06.java
class Ex06 {
    public static void main(String[] args) {
        int[][] M = {
            {1, 3, 4},
            {6, 9, 2},
            {-6, 8, 0},
            {5, 3, -2}
        };

        int R = M.length;
        int C = M[0].length;

        for (int r = 0; r < R; r++) {
            for (int c = 0; c < C; c++) {
                System.out.print(M[r][c]);
                if (c < C - 1) {
                    System.out.print(", ");
                }
            }
            System.out.print("\n");
        }
    }
}

File: general/matlab/as06/ex06.m
I1 = imread('ex_06.jpg');
I2 = imread('ex_03.gif');

I1 = rgb2gray(I1);
I2 = rgb2gray(I2);

I2 = imresize(I2, size(I1));

I3 = I2 - I1;

imshow(I3);